django_migration_linter/migration_linter.py

```diff
@@ -463,11 +463,10 @@ def lint_runpython(self, runpython):
                 warning.append(issue)
 
         # Detect warning for argument naming convention
-        args_spec = (
-            inspect.getargspec(runpython.code)
-            if PY2
-            else inspect.getfullargspec(runpython.code)
-        )
+        if sys.version_info[0] == 2:  # PY2
+            args_spec = inspect.getargspec(runpython.code)
+        else:
+            args_spec = inspect.getfullargspec(runpython.code)
```

Comment:
It could also try `getfullargspec`, and fall back to `getargspec` on AttributeError instead.
What do you think?